app/assets/javascripts/discourse/app/controllers/tag-show.js

```diff
@@ -109,11 +110,20 @@ export default Controller.extend(BulkTopicSelection, FilterModeMixin, {
     return this.isFilterPage(filter, "unread") && topicsLength > 0;
   },
 
+  @discourseComputed("list.filter", "list.topics.length")
+  showResetNew(filter, topicsLength) {
+    return this.isFilterPage(filter, "new") && topicsLength > 0;
```

Comment:
not sure about this quite yet, you could be dismissing new in the "dev" category tagged "docker". Can we add support for that in this PR as well?

Reply:
you need to enable a site setting to get the extra filter btw.

Reply:
ahh, I got you, you are right. If only a tag is selected it would be fine but we can have both category and tag selected. I will fix it